packages/app-store/routing-forms/trpc/utils.ts

```diff
@@ -18,6 +22,18 @@ import type { SerializableField, OrderedResponses } from "../types/types";
 import type { FormResponse, SerializableForm } from "../types/types";
 import { acrossQueryValueCompatiblity } from "./raqbUtils";
 
+let tasker: Tasker;
+
+if (typeof window === "undefined") {
+  import("@calcom/features/tasker")
+    .then((module) => {
+      tasker = module.default;
+    })
+    .catch((error) => {
+      console.error("Failed to load tasker:", error);
+    });
+}
```

Comment:
Is there a better way to import this? A normal import caused this issue: 
![Screenshot 2024-10-16 at 1 14 25 PM](https://github.com/user-attachments/assets/0dca35ee-5bba-4db5-9938-efe4d7e832cf)

Reply:
Tasker shouldn't be imported on utils that are used on client-side

Reply:
Pushed a fix

Reply:
reverted. It seems that `modules/apps/[slug]/[...pages]/pages-view.tsx` is using "use client" directive and we're importing server-side code in there. causing this mixup. We can fix in a follow up. cc @hbjORbj